src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java

```diff
@@ -358,19 +364,16 @@ private void doDispatch(@Nonnull Message message) {
             }
         }
     }
-    
-    private static final class SSEChannelSubscriber implements ChannelSubscriber {
-        
-        private EventDispatcher eventDispatcher;
-        private int numSubscribers = 0;
 
-        public SSEChannelSubscriber(EventDispatcher eventDispatcher) {
-            this.eventDispatcher = eventDispatcher;
-        }
+    /**
+     * Receive event from {@link PubsubBus} and sends it to this client.
+     */
+    private final class SSEChannelSubscriber implements ChannelSubscriber {
```

Comment:
I replaced the explicit `eventDispatcher` field by implicit `$0` so no impact on serialization.